### PR TITLE
Feature/atms

### PR DESCRIPTION
**Pull request by xyzemc**
This PR add the YAML file of satellite radiance ATMS_NPP into rrfs_test for MPAS_JEDI. This is based on the issue #32 temporally resolved the boundary check. The YAML file included the basic information of ATMS NPP instrument information, CRTM, and quality control information. The YAML file is: RDASApp//rrfs-test/testinput/rrfs_mpasjedi_2022052619_atms_npp_qc.yaml.

**The ctest result on Hera**:
[Xiaoyan.Zhang@hfe08 rrfs-test]$ ctest
Test project /scratch2/NCEPDEV/fv3-cam/Xiaoyan.Zhang/noscrub/JEDI/RDASApp_atms/build/rr
    Start 1: rrfs_mpasjedi_2022052619_Ens3Dvar
1/3 Test #1: rrfs_mpasjedi_2022052619_Ens3Dvar ......   Passed   71.91 sec
    Start 2: rrfs_mpasjedi_2022052619_letkf
2/3 Test #2: rrfs_mpasjedi_2022052619_letkf .........   Passed   37.02 sec
    Start 3: rrfs_mpasjedi_2022052619_atms_npp_qc
3/3 Test #3: rrfs_mpasjedi_2022052619_atms_npp_qc ...   Passed   89.96 sec

100% tests passed, 0 tests failed out of 3

Label Time Summary:
mpi            = 198.89 sec*proc (3 tests)
rdas-bundle    = 198.89 sec*proc (3 tests)
script         = 198.89 sec*proc (3 tests)

Total Test time (real) = 199.18 sec

### CODE REVIEW

File: modulefiles/RDAS/hera.intel.lua [hunk 1]
```diff
@@ -86,7 +86,8 @@ setenv('MPIEXEC_NPROC', mpinproc)
 setenv("CRTM_FIX","/scratch1/NCEPDEV/da/Cory.R.Martin/RDASApp/fix/crtm/2.4.0")
 setenv("RDASAPP_TESTDATA","/scratch1/NCEPDEV/da/Cory.R.Martin/CI/RDASApp/data")
 --setenv("RDAS_RRFS_DATA_ROOT","/scratch2/NCEPDEV/fv3-cam/Ting.Lei/dr-emc-rdas/")
-setenv("RDAS_RRFS_DATA_ROOT", "/scratch1/NCEPDEV/da/Samuel.Degelia/RRFS")
+--setenv("RDAS_RRFS_DATA_ROOT", "/scratch1/NCEPDEV/da/Samuel.Degelia/RRFS")
+setenv("RDAS_RRFS_DATA_ROOT", "/scratch1/NCEPDEV/da/Xiaoyan.Zhang/RRFS")
```
Comment (delippi) [inline modulefiles/RDAS/hera.intel.lua]: @ShunLiu-NOAA, it seems like we are getting to the point where a role account would be really helpful.

Comment (SamuelDegelia-NOAA) [inline modulefiles/RDAS/hera.intel.lua]: Do you want to send me the atms observation file to stage in my directory? Or we could use your directory. A role account would be helpful here like @ShunLiu-NOAA mentioned last week.

Comment (SamuelDegelia-NOAA) [reply modulefiles/RDAS/hera.intel.lua]: Whoops, Donnie commented on the same thing just now, so we can ignore this comment.

Comment (xyzemc) [reply modulefiles/RDAS/hera.intel.lua]: > Do you want to send me the atms observation file to stage in my directory? Or we could use your directory. A role account would be helpful here like @ShunLiu-NOAA mentioned last week.

I think either way is good.

**Comment by SamuelDegelia-NOAA** [comment]
Do we want to include reference output for this new test too, similar to the other ctests?

File: modulefiles/RDAS/hera.intel.lua (hunk 1 above)
Comment (ShunLiu-NOAA) [reply modulefiles/RDAS/hera.intel.lua]: I am working on the application of role account on Hera and Orion. Stay tuned. @xyzemc send data to @SamuelDegelia-NOAA for your test case. In future, test data will go to role account.

Comment (SamuelDegelia-NOAA) [reply modulefiles/RDAS/hera.intel.lua]: @xyzemc I just grabbed the `atms_npp_obs_2022052619.nc` from your staged rrfs-test-data directory. I will copy it into my directories on both Hera and Orion. Do you know if I need any other files else for this test case to run?

Comment (xyzemc) [reply modulefiles/RDAS/hera.intel.lua]: Don't forget the crtm coefficient crtm2.4.0.

Comment (SamuelDegelia-NOAA) [reply modulefiles/RDAS/hera.intel.lua]: Got it, everything should be included now. Could you change this line back to my directory?

**Comment by xyzemc** [comment]
> Do we want to include reference output for this new test too, similar to the other ctests?


Yes. 
The yaml file has been changed to include the reference output. The reference output file (rrfs-mpasjedi-atms_npp.ref) has been placed into /scratch2/NCEPDEV/fv3-cam/Xiaoyan.Zhang/noscrub/JEDI/RDASApp_atms/bundle/rrfs-test-data/rrfs-data_mpasjedi_2022052619/Data/testinput
The ctest result with this change is:

> 

> 

Test project /scratch2/NCEPDEV/fv3-cam/Xiaoyan.Zhang/noscrub/JEDI/RDASApp_atms/build/rrfs-test
  Test #1: rrfs_mpasjedi_2022052619_Ens3Dvar
  Test #2: rrfs_mpasjedi_2022052619_letkf
  Test #3: rrfs_mpasjedi_2022052619_atms_npp_qc

Total Tests: 3
[Xiaoyan.Zhang@hfe10 rrfs-test]$ ctest
Test project /scratch2/NCEPDEV/fv3-cam/Xiaoyan.Zhang/noscrub/JEDI/RDASApp_atms/build/rrfs-test
    Start 1: rrfs_mpasjedi_2022052619_Ens3Dvar
1/3 Test #1: rrfs_mpasjedi_2022052619_Ens3Dvar ......   Passed   80.14 sec
    Start 2: rrfs_mpasjedi_2022052619_letkf
2/3 Test #2: rrfs_mpasjedi_2022052619_letkf .........   Passed   40.22 sec
    Start 3: rrfs_mpasjedi_2022052619_atms_npp_qc
3/3 Test #3: rrfs_mpasjedi_2022052619_atms_npp_qc ...   Passed   87.70 sec

100% tests passed, 0 tests failed out of 3

Label Time Summary:
mpi            = 208.05 sec*proc (3 tests)
rdas-bundle    = 208.05 sec*proc (3 tests)
script         = 208.05 sec*proc (3 tests)

Total Test time (real) = 208.24 sec

File: modulefiles/RDAS/hera.intel.lua (hunk 1 above)
Comment (SamuelDegelia-NOAA) [reply modulefiles/RDAS/hera.intel.lua]: @xyzemc Thank you for adding the test reference data. Before merging, could you change the `RDAS_RRFS_DATA_ROOT` here in the `hera.intel.lua` back to my directory? That way we can keep everything in one place until we get access to the role account soon.

Comment (xyzemc) [reply modulefiles/RDAS/hera.intel.lua]: > @xyzemc Thank you for adding the test reference data. Before merging, could you change the `RDAS_RRFS_DATA_ROOT` here in the `hera.intel.lua` back to my directory? That way we can keep everything in one place until we get access to the role account soon.

Changed !

**Review by TingLei-NOAA — APPROVED** [review]

**Review by SamuelDegelia-NOAA — APPROVED** [review]

**Review by ShunLiu-NOAA — APPROVED** [review]

File: modulefiles/RDAS/hera.intel.lua (hunk 1 above)
Comment (ShunLiu-NOAA) [reply modulefiles/RDAS/hera.intel.lua]: The applications for the role account on Hera and Orion are submitted.

**Pull request merged**